src/common/Page.ts

```diff
@@ -1443,6 +1443,41 @@ export class Page extends EventEmitter {
     }
   }
 
+  /**
+   * Overrides the Idle state.
+   *
+   * @example
+   * ```js
+   * await page.setIdleOverride({isUserActive: true, isScreenUnlocked: false});
+   * ```
+   *
+   * @param isUserActive Mock isUserActive
+   * @param isScreenUnlocked Mock isScreenUnlocked
+   */
+  async setIdleOverride(
```

Comment:
What's the rationale for the current name? How would you feel about something like `emulateIdleState` (seems closer to existing Puppeteer APIs)? As an end user, this seems nice:

```js
page.emulateIdleState({ userActive: true, screenUnlocked: true });
```

Reply:
I'd also suggest to use the object instead of two boolean arguments.

Reply:
done